client/src/components/form/UploadAppFormStepper.jsx

```diff
@@ -139,19 +150,27 @@ const AppVersionSection = props => {
             <Field
                 name="minVer"
                 component={formUtils.renderSelectField}
-                label="Minimum DHIS version"
+                label="Minimum DHIS version *"
             >
                 {DHISVersionItems}
             </Field>
             <br />
             <Field
                 name="maxVer"
                 component={formUtils.renderSelectField}
-                label="Maximum DHIS version"
+                label="Maximum DHIS version *"
             >
                 {DHISVersionItems}
             </Field>
             <br />
+            <Field
+                name="channel"
+                component={formUtils.renderSelectField}
+                label="Release channel *"
+            >
```

Comment:
Should we have a default value?

Reply:
Personally I think it's better to require an active choice to not publish an app to the wrong channel, the same way we require selecting max/min version.

Reply:
I agree that an active choice would be OK.

More importantly, does this affect the App Management application when selecting apps for now or can it use the old way?

Reply:
The v1 "apps" api for fetching apps still just returns the stable apps. Currently no v2 api endpoint for fetching another release channel is implemented. Will merge this first and then  tweak this behaviour and add v2 endpoint for selecting a channel